tests/integration_tests/api/system_backend/test_quota.py

```diff
@@ -0,0 +1,135 @@
+import logging
+from unittest import TestCase
+
+from hvac import exceptions
+from tests.utils.hvac_integration_test_case import HvacIntegrationTestCase
+
+
+class TestQuota(HvacIntegrationTestCase, TestCase):
+    def setUp(self):
+        super().setUp()
+        self.client.sys.enable_secrets_engine(backend_type="kv", path="test")  # type: ignore
```

Comment:
Are these `# type: ignore` comments to address something in our project/linting? I don't think we need these.